sources/tests/workflow-docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     environment:
       - ZEEBE_BROKER_DATA_DISKUSAGECOMMANDWATERMARK=0.998
       - ZEEBE_BROKER_DATA_DISKUSAGEREPLICATIONWATERMARK=0.999
-      - "JAVA_TOOL_OPTIONS=-Xms256m -Xmx256m"
+      - "JAVA_TOOL_OPTIONS=-Xms512m -Xmx512m"
```

Comment:
Is it to reduce the memory consumption on the dev machine or GitHub action machines?
Can you please clarify which was the motivation around it?

Reply:
When I created the docker-compose exclusively for testing, provided 256m to keep it light. But considering the number of workflow scenarios now tested and will be tested in future, increased it to 512m to ensure stability of the zeebe.

Reply:
In the other PR I would comment on that. I don't know how it was possible to run that with only 256m 😄